lib/pwwka/configuration.rb

```diff
@@ -10,11 +11,12 @@ class Configuration
     attr_accessor :logger
     attr_accessor :log_level
     attr_accessor :options
-    attr_accessor :async_job_klass
+    attr_accessor :background_job_processor
     attr_accessor :send_message_resque_backoff_strategy
     attr_accessor :default_prefetch
     attr_reader   :requeue_on_error
     attr_writer   :app_id
+    attr_writer   :async_job_klass
```

Comment:
just as fyi since i looked it up, the checkout project seems to be the only one using this option: https://github.com/stitchfix/checkout/blob/79218a299265d539a0f867b09ee0502fe0f4c01f/config/initializers/pwwka.rb#L10